starport_template/lib/stores/wallets_store.dart

```diff
@@ -26,6 +26,8 @@ class WalletsStore {
   final Observable<bool> _isWalletImporting = Observable(false);
   final Observable<bool> _isWalletImportingError = Observable(false);
   final Observable<bool> _isBalancesLoadingError = Observable(false);
+  final Observable<WalletPublicInfo> _selectedWallet =
+      Observable(const WalletPublicInfo(chainId: '', name: '', publicAddress: '', walletId: ''));
```

Comment:
I couldn't figure out what should be the best way to make this non-nullable :/

Reply:
Awesome! 
There might not be a situation like that unless the app is installed for the first time. Otherwise, we will, by default, select the first wallet from the list of wallets.

Reply:
exactly. the app doesnt start if you have no wallets yet

Reply:
Yep, just fixed it. Now if there are no wallets, the app will start from onboarding automatically (which was already implemented in `routing_page.dart`), otherwise the first wallet will be automatically selected.